build-config/resources/Ballerina.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+org = "ballerina"
+name = "sql"
+version = "@toml.version@"
+distribution = "slbeta3"
+
+[[platform.java11.dependency]]
+path = "../native/build/libs/sql-native-@project.version@.jar"
+
+[[platform.java11.dependency]]
+path = "../test-utils/build/libs/sql-test-utils-@project.version@.jar"
+scope = "testOnly"
+
+[[platform.java11.dependency]]
+path = "./lib/HikariCP-3.3.1.jar"
```

Comment:
Shouldn't we handle these version numbers in the `updateTomlFile` task as well, and put the version in the `gradle.properties` file?

Reply:
These were not changed frequently, thus not templated so far.  Anyway fixed in 4b4c45c